src/components/interview.ts

```diff
@@ -35,177 +42,86 @@ function parseLink(link: string) {
   } else {
     return null;
   }
-}
-
-export async function handleInterview(message: Discord.Message, args: string[], client: Discord.Client): Promise<void> {
-  const command = args.shift();
-  switch (command) {
-    case 'signup':
-      addInterviewer(message, args);
-      break;
-    case 'list':
-      listInterviewers(message, client, args);
-      break;
-    case 'domain':
-      addDomain(message, args);
-      break;
-    case 'profile':
-      showProfile(message);
-      break;
-    case 'clear':
-      clearProfile(message);
-      break;
-    default:
-      help(message);
-      break;
-  }
-}
-
-async function help(message: Discord.Message): Promise<void> {
-  //should be deprecated as soon as proper help handler is built.
-  const response = new Discord.MessageEmbed()
-    .setTitle('Help')
-    .setColor('#0099ff')
-    .setDescription("I can't seem to recognize your command; the commands I know for interviewers are:")
-    .addFields(
-      { name: 'Interviewer Signup', value: '`.interviewer signup [calendly/xai link]`' },
-      { name: 'Get Interviewer List', value: '`.interviewer list (domain)`' },
-      { name: 'Add Interviewer Domain', value: '`.interviewer domain [domain]`' },
-      { name: 'Show Interviewer Profile', value: '`.interviewer profile`' },
-      { name: 'Clear Profile', value: '`.interviewer clear`' },
-      { name: 'Available Domains', value: '`' + getDomainString(available_domains) + '`' }
-    );
-  await message.reply(response);
-}
+};
 
-async function addInterviewer(message: Discord.Message, args: string[]): Promise<void> {
+/*
+  If interviewer doesn't exist, save the interviewer and their calendar link.
+  Otherwise, update the interviewer's calendar link.
+*/
+export const upsertInterviewer = async (id: string, calendarUrl: string): Promise<void> => {
```

Comment:
is upsert a thing?

Reply:
I've seen that word being used before and it's basically what we're trying to do https://en.wiktionary.org/wiki/upsert#:~:text=upsert%20(plural%20upserts),updates%20them%20if%20they%20do.

Reply:
oh it's a thing in postgres as well lol https://wiki.postgresql.org/wiki/UPSERT

Reply:
Hella weird terminology LOL